cli/main.rs

```diff
@@ -44,12 +44,14 @@ use std::env;
 use std::iter::once;
 use std::path::PathBuf;
 
-fn init_v8_flags(v8_flags: &[String]) {
-  let v8_flags_includes_help = v8_flags
+fn init_v8_flags(v8_flags: &[String], env_v8_flags: Vec<String>) {
+  let v8_flags_includes_help = env_v8_flags
     .iter()
+    .chain(v8_flags)
     .any(|flag| flag == "-help" || flag == "--help");
   // Keep in sync with `standalone.rs`.
   let v8_flags = once("UNUSED_BUT_NECESSARY_ARG0".to_owned())
+    .chain(env_v8_flags.into_iter())
     .chain(v8_flags.iter().cloned())
```

Comment:
Which one of these takes precedence? I think v8_flags should override the env flags? Maybe let's include this scenario in the integration test?

Reply:
in the help message I have written:
> Any flags set with this flag are appended after the DENO_V8_FLAGS environmental variable

should we do the opposite?

Reply:
however thats replacement, whereas in this case its extension. the question then also is, does the order of flags matter?

Reply:
Put into other words, if someone has a `DENO_V8_FLAGS` with `--histogram_interval=100000` then specifies a flag with `--historgram_interval=200000`, which value will be used? Based on looking at the code, I think it will be 100000, but I'm not sure. I think it should be the flag value of 200000.

Reply:
IIRC you can pass multiple flags with same name to V8 and it decides itself which one to use (the last one I believe)

Reply:
in that case, how it is implemented right now, the flag one would be used (sine its env then flags), so should be fine as-is

Reply:
Is there a way we could add a test to see if it's actually that way? That will also help us catch if v8 ever changes that behaviour for some reason (though highly unlikely).

Reply:
I dont think there is a way to get the v8 flags that the process was started with